src/sql/base/browser/ui/taskbar/taskbar.ts

```diff
@@ -130,6 +130,14 @@ export class Taskbar {
 		}
 	}
 
+	public addContent(content: ITaskbarContent): void {
```

Comment:
I commented later in the review that this method shouldn't be needed, since we should add the button on opening the notebook, but show as disabled / have different label until the model is loaded.

Reply:
Responded to other comment.